example/timer/odp_timer_accuracy.c

```diff
@@ -358,14 +364,21 @@ static int start_timers(test_global_t *test_global)
 		timer_param.max_tmo = test_global->opt.max_tmo_ns;
 	}
 
+	if (mode == 3) {
+		timer_param.timer_type = ODP_TIMER_TYPE_PERIODIC;
+		timer_param.periodic.base_freq_hz.numer = ODP_TIME_SEC_IN_NS;
+		timer_param.periodic.base_freq_hz.denom = period_ns;
+		timer_param.periodic.max_multiplier = 1;
```

Comment:
`odp_timer_periodic_capability()`should be used to check if parameters are supported.

Reply:
It could be simpler to add these values as new options, so we can tests exact values.